src/Select.js

```diff
@@ -82,6 +82,8 @@ export type Props = {
   'aria-labelledby'?: string,
   /* Focus the control when it is mounted */
   autoFocus?: boolean,
+  /* Focus first focusable option on menu open and when user is typing */
+  autoFocusFirstOption: boolean,
```

Comment:
I would maybe see something more like `autoFocusOption` with a value of `null`, `'first'`, or `'last'` for more flexibility.

Reply:
I will rename the prop, but 'first' or 'last' is decided in another part of the library, not the concern of this option.

Reply:
Ok fine by me then.